### DISCUSSION TITLE
Option to add title to svg to make key words searchable in GitHub

### DISCUSSION THREAD

**Discussion by SoumayaMauthoorMOJ** [excalidraw/excalidraw#8213] [Q&A]
I have created two svg-embedded diagrams using Excalidraw in this [test repo](https://github.com/SoumayaMauthoorMOJ/test-excalidraw):

- [box.excalidraw.svg](https://github.com/SoumayaMauthoorMOJ/test-excalidraw/blob/main/box.excalidraw.svg) contains a single box with the label "Presentations"  and a title tag set to "Presentations" which I have manually added 
- [box_without_title.excalidraw.svg](https://github.com/SoumayaMauthoorMOJ/test-excalidraw/blob/main/box_without_title.excalidraw.svg) contains the same box but no title

[Searching for the word "Presentations" in GitHub](https://github.com/search?q=repo%3ASoumayaMauthoorMOJ%2Ftest-excalidraw%20Presentations&type=code) only returns a match if included in the title tag.

Can I raise a feature to have the option to add title (and perhaps description) tags to images to make svg images easier to search?

Not a big rush since there's a dependency on fixing #4855 first to make sure SVG is truly compatible with GitHub

**Comment by SoumayaMauthoorMOJ** [comment]
Hello again 👋 Now that #4855 is fixed thanks to the amazing new Excalifont, I was wondering if I could raise a feature to better support svg search in GitHub?

**Reply by Mrazator** [reply]
Sure, let's track it in the following issue #8301. Since I am not sure when we could get to this, I've marked the issue as a `good first issue`, so possibly someone from the community could take it.
